features/items.md

```diff
@@ -0,0 +1,237 @@
+# Items
+
+## Introduction
+OpenHAB has a strict separation between the physical world (the “things”, see below) and the application, which is built around the notion of “items” (also called the virtual layer).
```

Comment:
is "things" the physical world? I think you may confuse readers with this statement. Things+Items are the abstract representation / the model of actual objects in the physical world.